apps/web/src/components/CommunityInterestDialog/DevelopmentProgramInterestItem.tsx

```diff
@@ -0,0 +1,190 @@
+import { useIntl } from "react-intl";
+import CheckCircleIcon from "@heroicons/react/20/solid/CheckCircleIcon";
+import ExclamationCircleIcon from "@heroicons/react/20/solid/ExclamationCircleIcon";
+import XCircleIcon from "@heroicons/react/20/solid/XCircleIcon";
+import QuestionMarkCircleIcon from "@heroicons/react/20/solid/QuestionMarkCircleIcon";
+import BuildingLibraryIcon from "@heroicons/react/20/solid/BuildingLibraryIcon";
+
+import {
+  DevelopmentProgramParticipationStatus,
+  FragmentType,
+  getFragment,
+  graphql,
+  Maybe,
+} from "@gc-digital-talent/graphql";
+import { commonMessages } from "@gc-digital-talent/i18n";
+import { IconType } from "@gc-digital-talent/ui";
+import { formatDate, parseDateTimeUtc } from "@gc-digital-talent/date-helpers";
+
+const hasEnrolled = (
+  status?: Maybe<DevelopmentProgramParticipationStatus>,
+): boolean => {
+  return (
+    !!status &&
+    [
+      DevelopmentProgramParticipationStatus.Enrolled,
+      DevelopmentProgramParticipationStatus.Completed,
+    ].includes(status)
+  );
+};
+
+interface StatusInfo {
+  Icon: IconType;
+  iconStyles: Record<string, string>;
+  message: string;
+}
+
+const useStatusInfo = (
+  status?: Maybe<DevelopmentProgramParticipationStatus>,
+  completionDate?: Maybe<string>,
+): StatusInfo => {
+  const intl = useIntl();
+
+  const defaultStatusInfo = {
+    Icon: ExclamationCircleIcon,
+    iconStyles: {
+      "data-h2-color": "base(error)",
+    },
+    message: intl.formatMessage(commonMessages.missingInformation),
+  };
+  if (!status || (!completionDate && hasEnrolled(status))) {
+    return defaultStatusInfo;
+  }
+
+  const date = completionDate
+    ? formatDate({
+        date: parseDateTimeUtc(completionDate),
+        formatString: "MMMM yyyy",
+        intl,
+      })
+    : intl.formatMessage(commonMessages.notAvailable);
+
+  const infoMap = new Map<DevelopmentProgramParticipationStatus, StatusInfo>([
+    [
+      DevelopmentProgramParticipationStatus.Interested,
+      {
+        Icon: QuestionMarkCircleIcon,
+        iconStyles: { "data-h2-color": "base(primary)" },
+        message: intl.formatMessage({
+          defaultMessage: "Interested in this program",
+          id: "ytcZ7A",
+          description:
+            "Message displayed when a user is interested in a development program",
+        }),
+      },
+    ],
+    [
+      DevelopmentProgramParticipationStatus.NotInterested,
+      {
+        Icon: XCircleIcon,
+        iconStyles: { "data-h2-color": "base(gray.lighter)" },
+        message: intl.formatMessage({
+          defaultMessage: "Not interested",
```

Comment:
This translation for this is `"Ce programme ne m'intéresse pas"` which is not the same as `"Not interested"`. @NienkeBr is this on purpose?